iphone/TitaniumKit/TitaniumKit/Sources/API/TiWindowProxy.h

```diff
@@ -28,6 +28,7 @@
   TiAnimation *closeAnimation;
   UIView *animatedOver;
   TiUIiOSTransitionAnimationProxy *transitionProxy;
+  UIModalPresentationStyle modalStyle;
```

Comment:
This seems to be used in `openOnUIThread:` only. Any specific reason this is an ivar instead of a local variable?

Reply:
This should be local. Updated.